openapi.yaml

```
@@ -2973,12 +2973,20 @@ paths:
       description: >-
         Calculates an estimate for a batch job. Back-ends can decide to
         either calculate the duration, the costs, the size or a combination of
-        them. This MUST be the upper limit of the incurring costs. Clients can
-        be charged less than specified, but never more. Back-end providers MAY
-        specify an expiry time for the estimate. Starting to process data
-        afterwards MAY be charged at a higher cost. Costs MAY NOT include
-        downloading costs. This can be indicated with the `downloads_included`
-        flag.
+        them.
+
+        Back-end providers MAY specify an expiry time for the estimate.
+        Starting to process data afterwards MAY be charged at a higher cost.
+        Costs MAY NOT include downloading costs. This can be indicated with the
```

What kind of downloading is meant here? Downloading raw/commercial data from a data source to the back-end or downloading the result from back-end to client?

---

"Downloading raw/commercial data from a data source to the back-end" isn't covered by the API at all yet, thus there's #447. So the initial thinking was pretty much about "downloading the result from back-end to client".

```suggestion
        Costs MAY NOT include downloading costs from the back-end to the client. This can be indicated with the
```

---

> downloading the result from back-end to client"

I'm not completely sure I understand in what sense a back-end could/would charge for this on the job level. You can somewhat estimate the cost of a job on itself, but you don't know how many times or what subset of assets a user will download.

---

Yes, that is why it is saying it here. This is the cost for processing, likely not for any download that may happen for the results, which may incur additional costs.

---

True, MAY NOT is indeed not part of RFC2119. What I meant to say is something like:

```suggestion
        Costs do often not include download costs. Whether download costs are included can be indicated with the
```